README.md

```diff
@@ -23,8 +23,20 @@ omit the title, it won't have a title.
 Write the body of your post below the "---". If you have headers in your post,
 use H2 and smaller headers (in makedown H2 is denoted by `##`).
 
+## Prerequisites
+
+To build the site, you will need [Jekyll](http://jekyllrb.com) version 2.5.3 or
+above. It can be installed with `apt-get install jekyll` on Ubuntu 16.10.
```

Comment:
It should work on Ubuntu 16.04 and might even work on older versions.
I think the text in its current form suggests that you need exactly 16.10 or newer.
Could you either clarify the version requirement or just leave out the version?

Reply:
Reworded, better?